lenet/README.md

```diff
@@ -83,8 +83,6 @@ Install dependencies:
 > npm install
```

Comment:
@huningxin Now it doesn't depend on the **mnist** node module, so please remove package.json and update the README.md, thanks.

Reply:
ditto